package.json

```diff
@@ -17,6 +17,8 @@
     "format": "lerna run format --stream",
     "format:docs": "prettier --write docs/**/*.html",
     "test:all": "lerna run test:all --stream",
+    "validate-spec": "npx swagger-cli validate docs/openapi/openapi.yml",
```

Comment:
shouldn't we add swagger-cli to the devDependencies?

Reply:
Just added it!